### DISCUSSION TITLE
[firebase_ml_vision] Add implementation for AutoML

### DISCUSSION THREAD

**Discussion by andrsdev** [firebase/flutterfire#5592] [Feature request]
**Description**
With the new feature of Firebase Auto ML it should be an implementation of it for Flutter.
Something similar to what is done in [Android](https://firebase.google.com/docs/ml-kit/android/label-images-with-automl?authuser=0#3.-run-the-image-labeler) and [IOS](https://firebase.google.com/docs/ml-kit/ios/label-images-with-automl?authuser=0)

**Comment by XinyueZ** [comment]
+1

**Comment by syeds-git** [comment]
Is this still in development? I can't find a way to get AuoML model working on device.
